sensor_msgs/JointState newState
---
int8 ret
